draft-ietf-tls-tls13.md

```diff
@@ -2234,7 +2211,7 @@ The "extension_data" field of this extension contains a
        enum {
            anonymous_RESERVED(0),
            rsa(1),
-           dsa_RESERVED(2),
+           dsa(2),
            ecdsa(3),
            rsapss(4),
```

Comment:
Yeah, I thought about that, but the script doesn't handle it well. Going to leave for now.